### PR TITLE
Adds async/await and more documentation to the heartbeat library

### LINKED ISSUE #1: Generate new project
As a user, I want a project generated for Skadi. 

It should just be a simple node library with all of the linting, CI, and configurations we have. 

This project will be the one we put in the a user's server.

### LINKED ISSUE #2: Send heartbeat information to skadi-endor
As a user, I want a heartbeat sent to skadi-endor. This heartbeat should contain a health check for the server.

### PR DESCRIPTION
**Pull request by clarkerican**
closes #1 Generate a new project AND
closes #2 Send heartbeat information to Koma

### CODE REVIEW

File: src/heartbeat/heartbeat.js [hunk 1]
```diff
@@ -8,16 +8,39 @@ let id = '';
 let serverUrl = '';
 let interval = TEN_MINUTES;
```
Comment (jackcmeyer) [inline src/heartbeat/heartbeat.js]: interval should be shorter I think, maybe a minute is better

Comment (nischayv) [reply src/heartbeat/heartbeat.js]: Waaaaaayyyyy shorter

Comment (nischayv) [reply src/heartbeat/heartbeat.js]: Once every second I would say. Apps can die and restart easily within a minute.

Comment (jackcmeyer) [reply src/heartbeat/heartbeat.js]: 💯

**Review by jackcmeyer — APPROVED** [review]
👍

**Review by nischayv — APPROVED** [review]

**Pull request merged**